.github/workflows/python_testing.yml

```diff
@@ -38,4 +38,3 @@ jobs:
           pytest-results-summary: true
         - macos: py3-xdist
           pytest-results-summary: true
-        - linux: py3-devdeps-xdist
```

Comment:
Is there a reason to remove this testing with devdeps? Isn't this what tests with dev versions of dependencies?

Reply:
This was testing with numpy 2.0 and I haven't started to look at that yet